packages/client/mutations/CreateTaskIntegrationMutation.ts

```diff
@@ -0,0 +1,184 @@
+import graphql from 'babel-plugin-relay/macro'
+import {stateToHTML} from 'draft-js-export-html'
+import {commitMutation} from 'react-relay'
+import {StandardMutation} from '../types/relayMutations'
+import splitDraftContent from '../utils/draftjs/splitDraftContent'
+import makeSuggestedIntegrationId from '../utils/makeSuggestedIntegrationId'
+import createProxyRecord from '../utils/relay/createProxyRecord'
+import {CreateTaskIntegrationMutation as TCreateTaskIntegrationMutation} from '../__generated__/CreateTaskIntegrationMutation.graphql'
+
+graphql`
+  fragment CreateTaskIntegrationMutation_task on CreateTaskIntegrationPayload {
+    task {
+      ...IntegratedTaskContent_task
+      integration {
+        __typename
+        ... on JiraIssue {
+          cloudId
+          cloudName
+          url
+          issueKey
+          summary
+          descriptionHTML
+          projectKey
+          project {
+            name
+          }
+        }
+        ... on _xGitHubIssue {
+          bodyHTML
+          title
+          number
+          repository {
+            nameWithOwner
+          }
+        }
+        ...TaskIntegrationLinkIntegrationGitHub
+        ...TaskIntegrationLinkIntegrationJira
```

Comment:
-1 without `@relay(mask: false)` I get the error
```
Error: Fragment TaskIntegrationLinkIntegrationJira was used, but not defined
    at githubRequest (/home/man/Work/parabol/parabol/dev/gqlExecutor.js:72097:28)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Promise.all (index 0)
    at async resolve (/home/man/Work/parabol/parabol/dev/gqlExecutor.js:49001:62)
    at async Object.TaskintegrationResolver (/home/man/Work/parabol/parabol/dev/gqlExecutor.js:33111:22) undefined
```

Looking just at the [docs](https://relay.dev/docs/api-reference/graphql-and-directives/#relaymask-boolean) it's not apparent why though. Also the documentation mentions `@inline` should be preferred instead, but I don't see the overlap from just the docs.
I'm just mentioning this as a learning opportunity, changing it to `@relay(mask: false)` should be good enough for this PR and @mattkrick will intervene otherwise.

Reply:
@Dschoordsch I don't see this error. How can I reproduce it?

Reply:
this is very strange because the stack trace is on the server & the `@relay` gets removed before sending the query to the server. There must be more here 🤔

Reply:
@igorlesnenko I had this error with only GitHub integration added and then pushing to GitHub. You're right, since it happened on the server, it looks like a relay bug.

Reply:
That's a weird one, the fragment is present when the error is thrown
```gql
mutation CreateTaskIntegrationMutation(                                                                                                                          
  $integrationProviderService: IntegrationProviderServiceEnum!                                                         
  $projectId: ID!
  $taskId: ID!                                                                                                         
) {                                                                                                                        
  createTaskIntegration(integrationProviderService: $integrationProviderService, projectId: $projectId, taskId: $taskId) {
    error {
      message
    }
    task {
      ...IntegratedTaskContent_task
      integration {
        __typename
        ... on JiraIssue {
          cloudId
          cloudName
          url
          issueKey
          summary
          descriptionHTML
          projectKey
          project {
            name
            id
          }
        }
        ... on _xGitHubIssue {
          bodyHTML
          title
          number
          repository {
            nameWithOwner
            id
          }
        }
        ...TaskIntegrationLinkIntegrationGitHub
        ...TaskIntegrationLinkIntegrationJira
        id
      }
      updatedAt
      teamId
      userId
      id
    }
  }
}
  
fragment IntegratedTaskContent_task on Task {
  integration {
    __typename
    ... on JiraIssue {
      descriptionHTML
      summary
    }
    ... on _xGitHubIssue {
      bodyHTML
      title
    }
    id
  }
}
    
fragment TaskIntegrationLinkIntegrationGitHub on _xGitHubIssue {
  number
  repository {
    nameWithOwner
    id
  }
}

fragment TaskIntegrationLinkIntegrationJira on JiraIssue {
  issueKey
  projectKey
  cloudName
}
```
so it seems something in the parsing is wrong. With `@relay(mask: false)` the fragments are inlined and the error disappears. There are still fragments present in that case, so I don't know what exactly the issue is.

Reply:
Added @relay(mask: false